doc/server/hap2/HowToUse.md

```diff
@@ -98,9 +98,16 @@ via pip with the following command:
 
 ### HAP2 Zabbix
 
+You need to install hatohol-hap2-zabbix with following command on CentOS 7:
+
+    # yum install -y hatohol-hap2-zabbix
+
 You should input `http://<servername or ip>/zabbix/api_jsonrpc.php` into
 "Zabbix API URL" instead of `<servername or ip>` simply.
 
+Also you have to input `amqp://<user>:<password>@hostname/<vhost>` into BrokerURL.
+These parameter should be replaced string that you input command for `$ sudo rabbitmqctl add_(user|vhost)`.
+
```

Comment:
Adding the following example broker URL note is better:

---

Note: 

If you follow this document instructions and both using hap2 plugins and hatohol server in same host, you can use `amqp://hatohol:hatohol@localhost/hatohol` into BrokerURL.